test/paper-spinner.html

```diff
@@ -54,6 +54,25 @@
             spinner.active = false;
             expect(spinner.getAttribute('aria-hidden')).to.be.eql('true');
           });
+
+          test('toggle during cooldown', function(done) {
+            spinner.active = true;
+
+            window.requestAnimationFrame(function() {
+              spinner.active = false;
+
+              // Set active to true before cooldown animation completes.
+              window.setTimeout(function() {
```

Comment:
Sure, I'll remove. For whatever reason this has become a habit when calling global functions..